Configuration/Geometry/README.md

```diff
@@ -5,31 +5,21 @@ Different versions of various subdetectors can be combined. The available versio
 Tracker:
 * T5: Phase2 tilted tracker (v6.1.3) w/ phase 2 pixel (v4.0.2.5) 
 * T6: Phase2 tilted tracker (v6.1.4) w/ phase 2 pixel (v4.0.4) (TEDD slighly rotated + Inner Tracker barrel has lower radii than TDR T5 geometry)
-* T11: Phase2 tilted tracker (v6.1.4) w/ phase 2 pixel (v4.0.5) (same as T6 but 50x50 in all modules)
```

Comment:
@emiglior @ghugo83  for my own understanding (in view of the other geometry-dependent conditions to be maintained for the future), is T11 (50x50um^2 cell) totally deprecated at this point? Is it going to be superseded by something else?

Reply:
@mmusich T11 is not yet totally deprecated as currently we do not have another geometry featuring square pixels. As you wrote, it will be superseded once we reach a more stable layout (e.g. skew TBPX ladders and MB updates)

Reply:
@emiglior thanks. But if I understand correctly the changes proposed here, T11 won't be exercised anymore in any production workflow (starting from the first pre-release in which this is merged). Is this the desired outcome? I am thinking in the scope of maintaining geometry-dependent IT conditions  bound to this layout.

Reply:
To be clear, T11 is *not* among the geometries explicitly listed to be kept in #26938, and therefore it is removed. We cannot maintain a monotonically increasing list of tracker geometries forever. The tracker group needs to set clear timelines and deliverables for alternate geometry studies.